src/cartservice/src/cartstore/RedisCartStore.cs

```diff
@@ -40,13 +40,13 @@ public RedisCartStore(string redisAddress)
             // Serialize empty cart into byte array.
             var cart = new Hipstershop.Cart();
             emptyCartBytes = cart.ToByteArray();
-            connectionString = $"{redisAddress},ssl=false,allowAdmin=true,connectRetry=5,abortConnect=false";
+            connectionString = $"{redisAddress},ssl=false,allowAdmin=true,abortConnect=false";
 
             redisConnectionOptions = ConfigurationOptions.Parse(connectionString);
 
-            // Try to reconnect if first retry failed (up to 5 times with exponential backoff)
```

Comment:
I have reworded the comment so we don't have to update it every time `REDIS_RETRY_NUM` changes.